modules/dnn/src/dnn.cpp

```diff
@@ -2575,11 +2919,17 @@ Net Net::readFromModelOptimizer(const String& xml, const String& bin)
 #ifndef HAVE_INF_ENGINE
     CV_Error(Error::StsError, "Build OpenCV with Inference Engine to enable loading models from Model Optimizer.");
 #else
+
+#if INF_ENGINE_VER_MAJOR_LE(INF_ENGINE_RELEASE_2019R3)
     InferenceEngine::CNNNetReader reader;
     reader.ReadNetwork(xml);
     reader.ReadWeights(bin);
 
     InferenceEngine::CNNNetwork ieNet = reader.getNetwork();
+#else
+    InferenceEngine::Core& ie = getCore();
+    InferenceEngine::CNNNetwork ieNet = ie.ReadNetwork(xml, bin);
```

Comment:
@dkurt Please review this part of code.

Reply:
@l-bat, Can we keep `ReadNetwork` and `ReadWeights` for now? I think it'd be more stable considering coming https://github.com/opencv/opencv/pull/16015.

Reply:
We can, but `CNNNetReader` is almost deprecated.